internal/datacoord/handler.go

```diff
@@ -148,13 +149,76 @@ func (h *ServerHandler) GetQueryVChanPositions(channel *channel, partitionID Uni
 		}
 	}
 
+	//var (
```

Comment:
just delete this

Reply:
it used to statistics the indexed segments, flushed segments and growing segments. Need QueryCoord to support this.